content/tutorials/manage-the-rippled-server/troubleshooting/server-wont-start.md

```diff
@@ -0,0 +1,169 @@
+
+- Edit your `rippled.cfg` file and modify the `[validators_file]` setting to have the correct path to your `validators.txt` (or equivalent) file. Check for extra whitespace before or after the filename.
+
+- Edit your `rippled.cfg` file and remove the `[validators_file]` setting. Add validator settings directly to your `rippled.cfg` file. For example:
+
+        [validator_list_sites]
+        https://vl.ripple.com
+
+        [validator_list_keys]
+        ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+
+
+## Cannot create database path
+
+If `rippled` crashes on startup with an error such as the following, it means the server does not have write permissions to the `[database_path]` from its config file.
+
+```text
+Loading: "/home/rippled/.config/ripple/rippled.cfg"
+Terminating thread rippled: main: unhandled St13runtime_error 'Can not create "/var/lib/rippled/db"'
+Aborted (core dumped)
+```
+
+The paths to the configuration file (`/home/rippled/.config/ripple/rippled.cfg`) and the database path (`/var/lib/rippled/db`) may vary depending on your system.
+
+Possible solutions:
+
+- Run `rippled` as a different user that has write permissions to the database path printed in the error message.
+
+- Edit your `rippled.cfg` file and change the `[database_path]` setting to use a path that the `rippled` user has write permissions to.
+
+- Grant the `rippled` user write permissions to the configured database path.
+
+
+## State DB Error
+
+The following error can occur if the `rippled` server's state database is corrupted (possibly as the result of being shutdown unexpectedly):
+
+```text
+2018-Aug-21 23:06:38.675117810 SHAMapStore:ERR state db error:
+  writableDbExists false archiveDbExists false
+  writableDb '/var/lib/rippled/db/rocksdb/rippledb.11a9' archiveDb '/var/lib/rippled/db/rocksdb/rippledb.2d73'
+
+To resume operation, make backups of and remove the files matching /var/lib/rippled/db/state* and contents of the directory /var/lib/rippled/db/rocksdb
+
+Terminating thread rippled: main: unhandled St13runtime_error 'state db error'
+```
+
+The easiest way to fix this problem is to delete the databases entirely. You may want to back them up elsewhere instead. For example:
+
+```sh
+mv /var/lib/rippled/db /var/lib/rippled/db-bak
+```
+
+Or, if you are sure you don't need the databases:
+
+```sh
+rm -r /var/lib/rippled/db
+```
+
+**Tip:** It is generally safe to delete the `rippled` databases, because any individual server can re-download ledger history from other servers in the XRP Ledger network. If you are using [clustering](clustering.html), be sure your servers each have a unique `[node_seed]` configured first; if not, servers may not be recognized as part of the cluster after you restart them.
```

Comment:
Should this be a caution or warning ... considering the part about unique `node_seed`?

Reply:
Better yet, I can just remove that part. The instructions for clustering explicitly tell you to generate and configure a `[node_seed]` so it's a pretty pointless warning to tell people that clustering won't work properly if you delete the databases while not using clustering properly.